jenkins-pipelines/upgrade_custom/rolling-upgrade-custom_d1.jenkinsfile

```diff
@@ -0,0 +1,14 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+rollingUpgradePipeline(
+    backend: 'gce',
+    base_versions: '',  // auto mode
+    linux_distro: 'ubuntu-focal',
+    use_preinstalled_scylla: true,
+    test_name: 'upgrade_test.UpgradeTest.test_custom_profile_rolling_upgrade',
+    test_config: '''["test-cases/upgrades/customer-profile/rolling-upgrade-custom-d1.yaml", "configurations/gce/n2-highmem-8.yaml"]''',
+    internode_compression: 'all',
```

Comment:
remove `internode_compression: 'all'`, we don't need it for this case

Reply:
removed